org.rdkit.knime.binaries.feature/feature.xml

```diff
@@ -2,15 +2,15 @@
 <feature
       id="org.rdkit.knime.binaries.feature"
       label="RDKit Binaries for Java"
-      version="3.6.0.qualifier"
+      version="3.6.1.qualifier"
       provider-name="NIBR">
 
    <description>
       This feature contains RDKit binaries for use as dependencies for other packages.
    </description>
 
    <copyright>
-      Copyright (C) 2015 Novartis Institutes for BioMedical Research
+      Copyright (C) 2019 Novartis Institutes for BioMedical Research
```

Comment:
I think it's better to do this as: `Copyright (C) 2015-2019` in order to indicate the full timeline.